torch/csrc/jit/passes/quantization/finalize.cpp

```diff
@@ -73,12 +73,14 @@ void FoldQuantizedPrepackingOps(Module& module) {
   PrePackingOpsFolder(module, filter_fn, "quantized");
 }
 
-Module Finalize(Module& module, QuantType quant_type) {
+Module Finalize(Module& module,
+                QuantType quant_type,
+                const std::vector<std::string>& preserved_attrs) {
```

Comment:
nit: are these subject to BC?

Reply:
you mean does this affect BC? I don't think so, these is an internal function.

for user function convert_jit this is an optional argument, so it won't affect BC either.